spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -1369,16 +1363,23 @@ def _do_load(self):
         # algorithms needed for loading the binaries to the SpiNNaker machine
         optional_algorithms.append("FrontEndCommonLoadExecutableImages")
 
+        # Get the executable targets
+        optional_algorithms.append("FrontEndCommonGraphBinaryGatherer")
```

Comment:
Should this come before the loading of the executables?  I know that the executor can reorder them (and probably will), but it saves my brain from hurting so much if it is in approximately the right order to start with...